src/test/java/org/folio/edge/sip2/api/support/BaseTest.java

```diff
@@ -1,5 +1,7 @@
 package org.folio.edge.sip2.api.support;
```

Comment:
For testing purposes, STunnel does not have to be installed on the developer's machine? If so can you please put a comment to this effect?

Reply:
For running unit tests/building, no. I was hoping it was pretty clear in the README that `stunnel` is a client side option for encrypted communication, but I hope it is not implied anywhere that `stunnel` is required. Do you think it is implied in the README?

I'd rather not confuse the issue by mentioning an SSL/TLS client in comments. It may bring about more questions, like what about `openssl` or some other wiz-bang SSL/TLS client. The unit tests use the vertx TCP client, which is SSL/TLS enabled.

If developers want to connect to a locally built edge-sip2 via SSL/TLS then some sort of encryption service will be needed. I'm not sure if this fact needs to be in a comment in the unit test though.

Reply:
It's clear that stunnel is a client side option, but I think a frequently asked question will be how to configure the developer's machine to test it, if required at all. 

> If developers want to connect to a locally built edge-sip2 via SSL/TLS then some sort of encryption service will be needed. I'm not sure if this fact needs to be in a comment in the unit test though.

Could it be mentioned in the README? After all this documentation is mostly for the developers.

Reply:
I updated the README with developer encryption details. Does this make it clear? For developers, it boils down to either, don't use it (the default) or set it up as described in the "Security" section.

Reply:
Yes, clearer now. Thanks.